### PR TITLE
Inclavare 2022 Annual Review

### PR DESCRIPTION
**Pull request by haosanzi**
This is [Inclavare's](https://github.com/inclavare-containers) annual sandbox review for 2022.

Signed-off-by: Shirong Hao <shirong@linux.alibaba.com>

### CODE REVIEW

File: reviews/2022-Inclavare-annual.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+With the help of CNCF, this project has gained more attention from the community and more developers have invested in the co-construction.
+
+This would help with the targets mentioned in `Goals`.
+
+# Incubation
+
+Now Inclavare Containers is not ready for incubation yet.
+
+# 参考
```
Comment (rootfs) [inline reviews/2022-Inclavare-annual.md:63]: `参考` -> `Reference`

File: reviews/2022-Inclavare-annual.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  - [deterministic builds](https://github.com/inclavare-containers/deterministic-builds): eBPF-based tool for deterministic builds.
+
+- Transparent network traffic gateway over attested channel using TEEs:
+  - [enclave network gateway](https://github.com/inclavare-containers/enclave-network-gateway)
+ 
+The developement work for the part of enclave container runtime will continue in [enclave-cc](https://github.com/confidential-containers/enclave-cc).
+
+# Help from CNCF
+
+One of our primary aims is to increase Inclavare Containers' visibility and adoption, together with share the view of process-level confidential containers to advance the Confidential Computing.
```
Comment (rootfs) [inline reviews/2022-Inclavare-annual.md:53]: `share` -> `sharing`

File: reviews/2022-Inclavare-annual.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+The DevStats statistics can be found [here](https://inclavarecontainers.devstats.cncf.io/d/1/activity-repository-groups?orgId=1&from=1631030400000&to=1662652799000&var-period=d7&var-repogroups=All).
+
+Note: 
+- The most of development efforts are contributed to the sub-projects under the organization https://github.com/inclavare-containers/ and [enclave-cc](https://github.com/confidential-containers/enclave-cc) rather than https://github.com/inclavare-containers/inclavare-containers.
+- The statistics for the development efforts on the part of enclave container runtime is moved to [here](https://github.com/confidential-containers/enclave-cc/graphs/contributors?from=2022-03-01&to=2022-11-08&type=c)
+
+# Maintainers
+
+The maintainer list can found [here](https://github.com/inclavare-containers/inclavare-containers/blob/master/MAINTAINERS.md).
```
Comment (rootfs) [inline reviews/2022-Inclavare-annual.md:19]: can be found

File: reviews/2022-Inclavare-annual.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+Inclavare Containers project doesn't just focus on container runtime for Intel SGX Enclave, it also creates several innovative technologies related to confidential container (See the `Goals` section for the details). Therefore, the most of development efforts on this project are contributed to these sub-projects under the organization https://github.com/inclavare-containers/.
+
+# DevStats
+
+The DevStats statistics can be found [here](https://inclavarecontainers.devstats.cncf.io/d/1/activity-repository-groups?orgId=1&from=1631030400000&to=1662652799000&var-period=d7&var-repogroups=All).
+
+Note: 
+- The most of development efforts are contributed to the sub-projects under the organization https://github.com/inclavare-containers/ and [enclave-cc](https://github.com/confidential-containers/enclave-cc) rather than https://github.com/inclavare-containers/inclavare-containers.
```
Comment (rootfs) [inline reviews/2022-Inclavare-annual.md:14]:
```text
Most of development efforts are contributed to the sub-projects under the organization [Inclavare Containers](https://github.com/inclavare-containers/)
```

File: reviews/2022-Inclavare-annual.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+The development efforts on the part of enclave container runtime was moved to [enclave-cc](https://github.com/confidential-containers/enclave-cc) under another CNCF project [Confidential Containers](https://github.com/confidential-containers) since 03/2022. The reason is that Inclavare Containers project can share [some common components](https://github.com/confidential-containers/enclave-cc/blob/main/docs/design.md#architecture) from Confidential Containers project, so this helps to reduce repetitive development efforts and keep active communication with neighbor community to collectively boost the adoption of confidential containers technology.
+
+Inclavare Containers project doesn't just focus on container runtime for Intel SGX Enclave, it also creates several innovative technologies related to confidential container (See the `Goals` section for the details). Therefore, the most of development efforts on this project are contributed to these sub-projects under the organization https://github.com/inclavare-containers/.
```
Comment (rootfs) [inline reviews/2022-Inclavare-annual.md:7]: try named anchor to cross reference the `Goals` section

**Comment by PushkarJ** [comment]
TOC, Please remove label for tag-security as we were unable to complete the review of the PR by Jan 1 deadline.

**Comment by amye** [comment]
Marking this as complete, annual reviews are no longer a requirement for sandbox. 
We'll be rolling out more efficient ways to show project health and metrics in 2024.

**Pull request merged**